gnomad_qc/v4/create_release/create_de_novo_release.py

```diff
@@ -99,16 +99,18 @@ def get_releasable_de_novo_calls_ht(
     return ht
 
 
-def aggregate_and_annotate_de_novos(ht: hl.Table) -> Tuple[hl.Table, hl.Table]:
+def aggregate_and_annotate_de_novos(ht: hl.Table) -> hl.Table:
     """
     Aggregate and annotate de novo calls.
 
-    This step get two sets of de novo calls:
-       - de novos at all confidence levels
-       - filtered high-confidence coding de novos
+    This step produces high-quality coding de novos for the release, including variants
+    that are not in low-confidence regions, do not have a `*` alt allele, and are not
+    excluded by variant QC A subset of filtered de novos, further restricted to HIGH
+    confidence or MEDIUM confidence with a P-value >= 0.9, coding consequence,
+    and gnomAD v4.1 exomes allele frequency (AF) and callset allele count (AC) filters.
```

Comment:
```suggestion
    This step produces high quality coding de novos for release. High quality is defined as variants that:
        - Are not in low-confidence regions
        - Do not have a `*` alt allele
        - Passed variant QC
        - Have either HIGH (all probabilities) or MEDIUM (P-value >=0.9) confidence of being a *de novo*
        - Have a coding consequence
        - Pass gnomAD v4.1 exomes allele frequency (AF) and callset allele count (AC) filters
```
optional reorganization -- I find this easier to read. can you double check I converted the clause about variant QC ("and are not excluded by variant QC A subset of filtered de novos") correctly?